src/main/java/racingGame/InputView.java

```diff
@@ -0,0 +1,17 @@
+package racingGame;
+
+import java.util.Scanner;
+
+public class InputView {
+    private final Scanner scanner = new Scanner(System.in);
+
+    public int readCarCount() {
```

Comment:
```suggestion
    public static int readCarCount() {
```
상태 값을 가지지 않는 객체의 메서드는 클래스 메서드(static)로 구현해 보면 어떨까?